### DISCUSSION TITLE
Trying quickstart - no module named habitat

### DISCUSSION THREAD

**Discussion by pranubs** [facebookresearch/habitat-lab#1433] [Q&A]
Hello, 

Im new to habitat. I installed habitat-sim and habitat-lab successfully using the instructions on the github page. Then I tested the installation by running the interavtive example - examples/viewer.py as given in the github readme page 
`python examples/viewer.py --scene /path/to/data/scene_datasets/habitat-test-scenes/skokloster-castle.glb`

This works perfectly. Next I wanted to try the example in the quickstart page : [https://aihabitat.org/docs/habitat-lab/quickstart.html](url)
But when I tried to run it inside my conda environment I get an error that I dont have the module habitat. I checked back in the viewer,py and the library used was habitat-sim that is 
`import habitat_sim` and not `import habitat`

Is there a special installation method to install 'habitat' library that i am missing? I checked out pip page for habitat : [https://pypi.org/project/habitat/](url) , but this seems like another library not related to habitat simulator. Am I missing something here? Or is there an error in the quickstart page? 
Any help will be appreciated. 
Thank you in advance!

**Comment by pranubs** [comment]
i had not installed habitat-lab, once i installed habitat-lab it worked

**Comment by EftePA** [comment]
As mentioned by @pranubs you need to install habitat-lab, but ensure your habitat-lab and habitat-sim have the same version. To check it use: `pip list | grep habitat`
